pos_partner_sale_warning/static/src/js/ProductScreen.esm.js

```diff
@@ -0,0 +1,28 @@
+/** @odoo-module **/
+
+import PartnerListScreen from "point_of_sale.PartnerListScreen";
+import Registries from "point_of_sale.Registries";
+
+export const PartnerListScreenWarning = (PartnerListScreen) =>
+    class extends PartnerListScreen {
+        clickPartner(partner) {
+            if (
+                this.state.selectedPartner &&
+                this.state.selectedPartner.id === partner.id
+            ) {
+                return super.clickPartner(partner);
+            }
+            if (partner && partner.sale_warn && partner.sale_warn !== "no-message") {
+                this.showPopup("ErrorPopup", {
+                    title: this.env._t(`Warning for ${partner.name}`),
```

Comment:
I'm not sure if it works, but it doesn't look safe anyway. Imagine that a translator replace code inside `${...}`.

```suggestion
                    title: this.env._t(`Warning for %(partner_name)s`, partner_name=partner.name),
```

Reply:
Resolved